### PR TITLE
Add blog for using the Open Liberty stack

### PR DESCRIPTION
**Pull request by awisniew90**

### CODE REVIEW

**Comment by awisniew90** [comment]
Fixes #868

**Comment by awisniew90** [comment]
Draft PRs - https://github.com/OpenLiberty/blogs/pull/941 / https://github.com/OpenLiberty/blogs/pull/950 / https://github.com/OpenLiberty/blogs/pull/952 / https://github.com/OpenLiberty/blogs/pull/953

File: posts/2020-12-11-open-liberty-devfile-stack.adoc [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+After link:https://odo.dev/docs/installing-odo/[installing odo], you can use the Open Liberty stack to either link:https://github.com/OpenLiberty/application-stack/wiki/Using-the-Default-Starter#getting-started[create a new Open Liberty starter application] or initialize an existing app. For this blog, we will be using the link:https://github.com/OpenLiberty/application-stack-intro[application-stack-intro] project as a starting point.
+
+
+[source,sh]
+----
+$ git clone git@github.com:OpenLiberty/application-stack-intro.git && \
+  cd application-stack-intro
+----
+
+This application is a basic JAX-RS sample using Maven, Open Liberty and Microprofile.
```
Comment (yeekangc) [inline posts/2020-12-11-open-liberty-devfile-stack.adoc]:
```suggestion
This application is a basic JAX-RS sample using Maven, Open Liberty and MicroProfile. 
```

**Review by yeekangc — DISMISSED** [review]

**Comment by yeekangc** [comment]
LGTM from a technical standpoint.

@awisniew90, it will be useful to mention "odo" somewhere in the beginning and/or in the title plus meta-data too?  May be a title like "Develop cloud-native Java apps directly in OpenShift with Open Liberty and odo"?

@GraceJansen, we should prioritize this high since this is for new capabilities that we made available recently.

Cc @Charlotte-Holt @dmuelle @scottkurz

**Comment by gljrobinson** [comment]
@yeekangc I've already prioritized this but unfortunately we will not be able to get this edited before Christmas due to vacation days, etc. 
The other item on the prioritized backlog also relates to features we recently released in our beta, so both are high priority.

**Comment by awisniew90** [comment]
@yeekangc - updated the title and fixed "MicroProfile"

**Comment by yeekangc** [comment]
Thank you all!

**Comment by awisniew90** [comment]
Hi @GraceJansen  - just wondering if you had an ETA for when this may get reviewed. Thanks!

**Comment by gljrobinson** [comment]
Hi @awisniew90, this has been assigned to @Charlotte-Holt for review and our targeted date for publish is the 20th Jan, although this is may change. Charlotte is aiming to review it this week.

**Review by Charlotte-Holt — APPROVED** [review]
ID reviewed and edited. Looks good. Merging to staging to validate.

**Pull request merged**